frontend/sw360-portlet/src/main/webapp/html/components/detailRelease.jsp

```diff
@@ -58,26 +58,52 @@
     <link rel="stylesheet" href="<%=request.getContextPath()%>/css/sw360.css">
     <script type="text/javascript" src="<%=request.getContextPath()%>/js/releaseTools.js"></script>
 
-    <div id="header"></div>
-    <p class="pageHeader"><label id="releaseHeaderLabel"><span class="pageHeaderBigSpan">Component: <sw360:out value="${component.name}"/></span>
-        <select id="releaseSelect" onchange="this.options[this.selectedIndex].value
-        && (window.location = createDetailURLfromReleaseId (this.options[this.selectedIndex].value) );">
-            <core_rt:forEach var="releaseItr" items="${component.releases}">
-                <option <core_rt:if test="${releaseItr.id == releaseId}"> selected </core_rt:if>
-                        value="${releaseItr.id}"><sw360:ReleaseName release="${releaseItr}" />
-                </option>
-            </core_rt:forEach>
-        </select>
-        <span class="pull-right">
-        <input type="button" id="edit" data-release-id="${releaseId}" value="Edit" class="addButton">
-        <sw360:DisplaySubscribeButton email="<%=themeDisplay.getUser().getEmailAddress()%>" object="${release}" id="SubscribeButton" />
-    </span>
-    </label>
-    </p>
+      <div id="header">
+        <p class="pageHeader"> <span class="pageHeaderBigSpan"> Component: ${component.name}</span>
+
+            <span class="dropdown">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <core_rt:if test="${releaseItr.id == releaseId}"> <button onclick="clickOnDropdown()" class="dropbtn"> ${releaseItr.name} ${releaseItr.version} <span class="arrow-down"/>  </button> </core_rt:if>
+                </core_rt:forEach>
+                <span id="releaseDropdown" class="dropdown-content">
+                <core_rt:forEach var="releaseItr" items="${component.releases}">
+                    <a href='<portlet:renderURL ><portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/><portlet:param name="<%=PortalConstants.RELEASE_ID%>" value="${releaseItr.id}"/><portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_RELEASE_DETAIL%>"/></portlet:renderURL>' target='_self'>${releaseItr.name} ${releaseItr.version}
+                        <core_rt:if test="${releaseItr.clearingState.value == 4}">  <span class="sw360CircleOK"> </span> </core_rt:if>
```

Comment:
- maybe extract this logic to a function instead of copy paste each line.
(mapping clearing key to class)

- we should think about configurability and maintainability here.
Currently it is really hard coded on the fronted

- release entries should be created even if the clearing state is not mapped to specific circle 
(e.g. a new clearing State (value 5) should be visible in the list without a circle)

Reply:
release entries will be created even if the clearing state is not mapped to a specific circle.